modules/distr-tracing-tempo-object-storage-setup-azure-sts-install.adoc

```diff
@@ -0,0 +1,126 @@
++
+[TIP]
+====
+You can get the URL of the OpenID Connect (OIDC) issuer for your cluster by running the following command:
+----
+$ oc get authentication cluster -o json | jq -r .spec.serviceAccountIssuer
+----
+====
+
+. Create a federated identity credential for the Kubernetes service account for use by the Query Frontend component of the {TempoShortName}:
++
+[source,terminal]
+----
+$ az identity federated-credential create \ # <1>
+  --name <credential_name>-frontend \ # <2>
+  --identity-name <identity_name> \
+  --resource-group <resource_group> \
+  --issuer <cluster_issuer> \
+  --subject <tempo_service_account_query_frontend_subject> \ # <3>
+  --audiences <audience> | jq
+----
+<1> Federated identity credentials allow Kubernetes service accounts to authenticate as an Azure managed identity without storing secrets or using an Azure service principal identity.
+<2> Choose a name for the frontend federated identity credential.
+<3> The service account subject for your cluster in the following format: `system:serviceaccount:<namespace>:tempo-<tempostack_instance_name>`.
+
+. Assign the Storage Blob Data Contributor role to the Azure service principal identity of your created Azure managed identity:
++
+[source,terminal]
+----
+$ az role assignment create \
+  --assignee <assignee_name> \ # <1>
+  --role "Storage Blob Data Contributor" \
+  --scope "/subscriptions/<subscription_id>
+----
+<1> The Azure service principal identity of the Azure managed identity that you created in step 1.
++
+[TIP]
+====
+You can get the `<assignee_name>` value by running the following command:
+----
+$ az ad sp list --all --filter "servicePrincipalType eq 'ManagedIdentity'" | jq -r --arg idName <identity_name> '.[] | select(.displayName == $idName) | .appId'`
+----
+====
+
+. Fetch the client ID of the Azure managed identity that you created in step 1:
++
+[source,bash]
+----
+CLIENT_ID=$(az identity show \
+  --name <identity_name> \ # <1>
+  --resource-group <resource_group> \ # <2>
+  --query clientId \
+  -o tsv)
+----
+<1> Copy and paste the `<identity_name>` value from step 1.
+<2> Copy and paste the `<resource_group>` value from step 1.
+
+. Create a Kubernetes secret for the Azure workload identity federation (WIF):
++
+[source,terminal]
+----
+$ oc create -n <tempo_namespace> secret generic azure-secret \
+  --from-literal=container=<azure_storage_azure_container> \ # <1>
+  --from-literal=account_name=<azure_storage_azure_accountname> \ # <2>
+  --from-literal=client_id=<client_id> \ # <3>
+  --from-literal=audience=<audience> \ # <4>
+  --from-literal=tenant_id=<tenant_id> # <5>
+----
+<1> The name of the Azure Blob Storage container.
+<2> The name of the Azure Storage account.
+<3> The client ID of the managed identity that you fetched in the previous step.
+<4> Optional: Defaults to `api://AzureADTokenExchange`.
+<5> Azure Tenant ID.
```

Comment:
To use the created Azure credential, we should also document the TempoStack and TempoMonolithic CR, for example. 
```
apiVersion: tempo.grafana.com/v1alpha1
kind: TempoStack
metadata:
  name: azurewiftm
  namespace: chainsaw-azurewif-tempo
spec:
  storage:
    secret:
      name: azure-secret
      type: azure

apiVersion: tempo.grafana.com/v1alpha1
kind: TempoMonolithic
metadata:
  name: azurewifmn
  namespace: chainsaw-azurewif-mono
spec:
  storage:
    traces:
      backend: azure
      azure:
        secret: azure-secret
```